### ISSUE TITLE
Exception thrown calling findJavaHome on OSX

### ISSUE THREAD

**Issue by YodaDaCoda** [jsdevel/node-find-java-home#34]
Source appears to be some trouble resolving symlinks. I've freshly installed openjdk@11 using homebrew.

https://github.com/jsdevel/node-find-java-home/blob/ed82ff45752b6bdbc826a56cf19c70d436280514/src/index.ts#L106

This appears to happen when getting the realpath of `/usr/local/opt/openjdk@11/bin/javac` which points to `../libexec/openjdk.jdk/Contents/Home/bin/javac`. This should resolve to `/usr/local/opt/openjdk@11/libexec/openjdk.jdk/Contents/Home/bin/javac`, which does exist.

```
node:internal/fs/utils:343
    throw err;
    ^

Error: ENOENT: no such file or directory, lstat '../libexec/openjdk.jdk/Contents/Home/bin/javac'
    at Object.lstatSync (node:fs:1504:3)
    at findLinkedFile (/Users/will/butn-salesforce/node-java-connector/node_modules/find-java-home/dist/index.js:257:13)
    at findLinkedFile (/Users/will/butn-salesforce/node-java-connector/node_modules/find-java-home/dist/index.js:259:12)
    at /Users/will/butn-salesforce/node-java-connector/node_modules/find-java-home/dist/index.js:155:24
    at /Users/will/butn-salesforce/node-java-connector/node_modules/which/which.js:52:18
    at FSReqCallback.oncomplete (node:fs:196:5) {
  errno: -2,
  syscall: 'lstat',
  code: 'ENOENT',
  path: '../libexec/openjdk.jdk/Contents/Home/bin/javac'
}
```

**Comment by jsdevel** [comment]
@YodaDaCoda please submit a PR

**Comment by YodaDaCoda** [comment]
Not sure what's going on here. The more I started digging, the more I thought it was an issue with the standard node `fs` library. But when I cloned the `node-java-home` repo and built it using `npx tsc`, I saw the output was different to what's published on npm. When I copy-pasted the generated `index.js` into my project's `node_modules` folder, the error went away and the project functioned as expected. I note that the published `index.js` has a function called `findLinkedFile`, which is the function in which the error occurs. I can only guess this function has been imported from `fs` by `tsc`, but I'm left confused as to why the output on my machine is different to what's published.

Apologies for the stream of consciousness, I'm just not sure where to go from here.
